src/python/doc/cubical_complex_tflow_itf_ref.rst

```diff
@@ -0,0 +1,40 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+TensorFlow layer for cubical persistence
+########################################
+
+.. include:: differentiation_sum.inc
+
+Example of gradient computed from cubical persistence
+-----------------------------------------------------
+
+.. testcode::
+
+    from gudhi.tensorflow import *
```

Comment:
Maybe to be reworked according to [this discussion](https://github.com/GUDHI/gudhi-devel/pull/411#discussion_r742337781)
```suggestion
    from gudhi.tensorflow import CubicalLayer
```

Reply:
Thanks! Done.